src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderValue.cs

```diff
@@ -46,6 +64,11 @@ private static ReadOnlyMemory<byte> Encode(CborWriter writer)
             return buffer.AsMemory();
         }
 
+        /// <summary>
+        /// Creates a <see cref="CoseHeaderValue"/> instance from a signed integer."/>.
+        /// </summary>
+        /// <param name="value">The signed integer to be encoded and that the instance will represent.</param>
+        /// <returns>An instance that represents the CBOR encoded <paramref name="value"/>.</returns>
```

Comment:
Again, encoding doesn't necessarily happen here.

```suggestion
        /// <returns>An instance that represents specified value.</returns>
```